cocotb/triggers.py

```diff
@@ -554,6 +556,18 @@ class First(_AggregateWaitable):
             t2 = Timer(10, units='ps')
             t_ret = yield First(t1, t2)
     """
+    @staticmethod
+    @decorators.coroutine
+    def __internal_waiter(t, e, completed):
```

Comment:
In the previous code, `t` and `e` were pretty obvious, but with it being a separate function could we go for more descriptive names (event, trigger, completed_triggers)?

Reply:
Seems reasonable, will wait for performance numbers before changing